backend/backoffice/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from backoffice.models import TalkEntry
+
+
+@admin.register(TalkEntry)
+class TalkEntryAdmin(admin.ModelAdmin):
+    search_fields = ("name", "title")

```

Comment:
Let's also add the future field `speakers` here, so that we can search by speaker name.

I think we can remove `name`, as it's just a slug representing the `title`. We, humans, will probably use only the `title` : )

Reply:
Add speakers and title